### PR TITLE
Spanish Translation

### PR DESCRIPTION
**Pull request by wowficcion**

### CODE REVIEW

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 1]
```diff
@@ -0,0 +1,816 @@
...
+#, fuzzy, python-brace-format
+#| msgid ""
+#| "{1}\n"
+#| "        _  _ . ___  ___  ___ _  _  {0}___ _  _ ___{1}\n"
+#| "        |__| | ]  | ]  | |__ |\\ |  {0}|__ \\__/ |__{1}\n"
+#| "        |  | | ]__| ]__| |__ | \\|  {0}|__  ||  |__{1}\n"
+#| "\n"
+#| "                    {0}[{1}!{0}]{1} Network error. Verify your connection.\n"
+#| "\n"
+msgid ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Seems wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 2]
```diff
@@ -0,0 +1,816 @@
...
+" {0}[{1}!{0}]{0}SERVEO IS MORE ADVANCED THAN NGROK.\n"
+" {0}[{1}!{0}]{0}STILL ITS YOUR CHOICE{1}"
+msgstr ""
+
+#: Actions.py:89
+#, fuzzy, python-brace-format
+msgid ""
+"\n"
+" {0}Please select any available server:{1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: All okay here..

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 3]
```diff
@@ -0,0 +1,816 @@
...
+"        {0}http://github.com/darksecdevelopers\n"
+"        {0}** BY:DARKSEC ** \n"
+"\n"
+"-------------------------------\n"
+"{0}[ SERVEO URL TYPE SELECTION ]{1}!! {0}\n"
+"-------------------------------\n"
+"{0}[{1}!{0}]{1}REMEMBER ? Custom Url's generating Can irritate you.\n"
+"{0}[{1}!{0}]{1}As the Serveo becomes popular, Getting a desired domain may be "
+"tough. "
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 4]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Actions.py:127
+#, python-brace-format
+msgid ""
+"\n"
+"{0}[{1}*{0}]{0}CHOOSE ANY SERVEO URL TYPE TO GENERATE PHISHING LINK:{1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 5]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Actions.py:128
+#, python-brace-format
+msgid ""
+"\n"
+"{0}[{1}1{0}]{1}Custom URL {0}(Generates desired url) \n"
+"{0}[{1}2{0}]{1}Random URL {0}(Generates Random url)"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 6]
```diff
@@ -0,0 +1,816 @@
...
+"-------------------------------\n"
+"{0}[ CREATE A CUSTOM URL HERE ]{1}!! {0}\n"
+"-------------------------------\n"
+"\n"
+"{0}[{1}!{0}]{1}[ YOU CAN MAKE YOUR URL SIMILAR TO AUTHENTIC URL's ]\n"
+"{0}[{1}!{0}]{1}Again ? Use Your Social-engineering Skills Here.\n"
+"\n"
+"\n"
+"{0}Insert a custom subdomain for serveo"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 7]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Actions.py:148
+#, python-brace-format
+msgid ""
+"\n"
+"{0}CUSTOM Subdomain>>> {1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Ok

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 8]
```diff
@@ -0,0 +1,816 @@
...
+"\n"
+"\n"
+"{0}FAILED TO GET THIS DOMAIN. !!!\n"
+"\n"
+"{0}LOOKS LIKE CUSTOM URL IS NOT VALID or ALREADY OCCUPIED BY SOMEONE "
+"ELSE. !!!\n"
+"\n"
+"{0}[{1}!{0}]TRY TO SELECT ANOTHER CUSTOM DOMAIN{1} (GOING BACK).. !! \n"
+"\n"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 9]
```diff
@@ -0,0 +1,816 @@
...
+"        {1}_  _ . ___  ___  ___ _  _  {0}___ _  _ ___{1}\n"
+"        |__| | ]  | ]  | |__ |\\ |  {0}|__ \\__/ |__{1}\n"
+"        |  | | ]__| ]__| |__ | \\|  {0}|__  ||  |__{1}\n"
+"        {0}http://github.com/darksecdevelopers\n"
+"        {0}** BY:DARKSEC ** \n"
+"\n"
+"-------------------------------\n"
+"{0}[ RANDOM SERVEO URL ]{1}!! {0}\n"
+"-------------------------------"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 10]
```diff
@@ -0,0 +1,816 @@
...
+"        _  _ . ___  ___  ___ _  _  {0}___ _  _ ___{1}\n"
+"        |__| | ]  | ]  | |__ |\\ |  {0}|__ \\__/ |__{1}\n"
+"        |  | | ]__| ]__| |__ | \\|  {0}|__  ||  |__{1}\n"
+"\n"
+"                    {0}[{1}!{0}]{1} Error al conectar. Verifica tu conexion.\n"
+"\n"
+
+#: Actions.py:193
+msgid "[PHP INSTALLATION FOUND]"
+msgstr "[PHP INSTALADO]"
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: I'm not sure about it, seems like ok

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 11]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Actions.py:196
+#, python-brace-format
+msgid ""
+" --{0}>{1} PHP NOT FOUND: \n"
+" {0}*{1} Please install PHP and run me again.http://www.php.net/"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 12]
```diff
@@ -0,0 +1,816 @@
...
+" --{0}>{1} PHP NO ENCONTRADO: \n"
+" {0}*{1} Por favor instala PHP y ejecutame otra vez.http://www.php.net/"
+
+#: Actions.py:202
+#, python-brace-format
+msgid "{0}[{1}*{0}]{1} HiddenEye is Opening. Please Wait...{2}%"
+msgstr "{0}[{1}*{0}]{1} HiddenEye esta cargando. Por favor espera...{2}%"
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 13]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Actions.py:205
+#, python-brace-format
+msgid ""
+"\n"
+"{2}[{1}!{2}]{1} Do you agree to use this tool for educational purposes only? "
+"({2}y{1}/{0}n{1})\n"
+"{2}HiddenEye >>> {1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 14]
```diff
@@ -0,0 +1,816 @@
...
+" {0}[{1}8{0}]{1} Instagram\n"
+" {0}[{1}9{0}]{1} Snapchat\n"
+" {0}[{1}10{0}]{1} Yahoo\n"
+" {0}[{1}11{0}]{1} Twitch\n"
+" {0}[{1}12{0}]{1} Microsoft\n"
+" {0}[{1}13{0}]{1} Steam\n"
+" {0}[{1}14{0}]{1} VK\n"
+" {0}[{1}15{0}]{1} iCloud\n"
+"{0}HiddenEye >>>  {1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Too much "\n"

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 15]
```diff
@@ -0,0 +1,816 @@
...
+"        {1}_  _ . ___  ___  ___ _  _  {0}___ _  _ ___{1}\n"
+"        |__| | ]  | ]  | |__ |\\ |  {0}|__ \\__/ |__{1}\n"
+"        |  | | ]__| ]__| |__ | \\|  {0}|__  ||  |__{1}\n"
+"        {0}http://github.com/darksecdevelopers\n"
+"        {0}** BY:DARKSEC ** \n"
+"\n"
+"-------------------------------\n"
+"{0}[ PUT YOUR REDIRECTING URL HERE ]{1}!! {0}\n"
+"-------------------------------"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 16]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Actions.py:307
+#, fuzzy, python-brace-format
+#| msgid ""
+#| "\n"
+#| " (Choose Wisely As Your Victim Will Redirect to This Link)"
+msgid ""
+"\n"
+"{1}**{0}(Choose Wisely As Your Victim Will Redirect to This Link)"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Seems like wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 17]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Actions.py:308
+#, fuzzy, python-brace-format
+#| msgid ""
+#| "\n"
+#| " (Leave Blank To Loop The Phishing Page)"
+msgid ""
+"\n"
+"{1}**{0}(Leave Blank To Loop The Phishing Page)"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 18]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Actions.py:309
+#, fuzzy, python-brace-format
+#| msgid ""
+#| "\n"
+#| " {0}Insert a custom redirect url:"
+msgid ""
+"\n"
+"{0}[{1}/!\\{0}]{0}Insert a custom redirect url:"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 19]
```diff
@@ -0,0 +1,816 @@
...
+"\n"
+" {0}Coloca una URL:"
+
+#: Actions.py:310
+#, python-brace-format
+msgid ""
+"\n"
+"{0}REDIRECT HERE>>> {1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: ok

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 20]
```diff
@@ -0,0 +1,816 @@
...
+#| "                    {0}[{1}!{0}]{1} Network error. Verify your connection.\n"
+#| "\n"
+msgid ""
+"{1}\n"
+"        _  _ . ___  ___  ___ _  _  {0}___ _  _ ___{1}\n"
+"        |__| | ]  | ]  | |__ |\\ |  {0}|__ \\__/ |__{1}\n"
+"        |  | | ]__| ]__| |__ | \\|  {0}|__  ||  |__{1}\n"
+"        {0}http://github.com/darksecdevelopers\n"
+"        {0}** BY:DARKSEC **\n"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 21]
```diff
@@ -0,0 +1,816 @@
...
+"                    {0}[{1}!{0}]{1} Error al conectar. Verifica tu conexion.\n"
+"\n"
+
+#: Actions.py:344
+#, python-brace-format
+msgid ""
+"-------------------------------\n"
+"{0}[ KEYLOGGER PROMPT ]{1}!! {0}\n"
+"-------------------------------"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 22]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Actions.py:347
+#, python-brace-format
+msgid ""
+"\n"
+"{0}[{1}*{0}]{0}DO YOU WANT TO ADD A KEYLOGGER IN PHISHING PAGE-{1}(Y/N)"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 23]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Actions.py:359
+#, python-brace-format
+msgid ""
+"\n"
+"{0}[{1}#{0}]Keylgger{0} ADDED !!!"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 24]
```diff
@@ -0,0 +1,816 @@
...
+"{0}[{1}*{0}]{1} Waiting for credentials//Keystrokes//Victim's device info. \n"
+msgstr ""
+"{0}[{1}*{0}]{1} Esperando por contraseñas//Keystrokes de las Victimas. \n"
+
+#: Actions.py:401
+#, python-brace-format
+msgid ""
+" {0}[ CREDENTIALS FOUND ]{1}:\n"
+" {0}{2}{1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 25]
```diff
@@ -0,0 +1,816 @@
...
+" {0}[ CONTRASEÑAS ENCONTRADAS ]{1}:\n"
+" {0}{2}{1}"
+
+#: Actions.py:417
+#, python-brace-format
+msgid ""
+" \n"
+"{0}[ VICTIM IP BONUS ]{1}:\n"
+" {0}{2}{1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 26]
```diff
@@ -0,0 +1,816 @@
...
+"{0}[ INFORMACION DE LA VICTIMA ENCONTRADA ]{1}:\n"
+" {0}{2}{1}"
+
+#: Actions.py:424
+#, python-brace-format
+msgid ""
+" \n"
+"{0}Longitude: {2} \n"
+"Latitude: {3}{1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 27]
```diff
@@ -0,0 +1,816 @@
...
+"{0}Longitud: {2} \n"
+"Latitud: {3}{1}"
+
+#: Actions.py:425
+#, python-brace-format
+msgid ""
+" \n"
+"{0}ISP: {2} \n"
+"Country: {3}{1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 28]
```diff
@@ -0,0 +1,816 @@
...
+"{0}ISP: {2} \n"
+"PAIS: {3}{1}"
+
+#: Actions.py:426
+#, python-brace-format
+msgid ""
+" \n"
+"{0}Region: {2} \n"
+"City: {3}{1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 29]
```diff
@@ -0,0 +1,816 @@
...
+" \n"
+"{0}Region: {2} \n"
+"Ciudad: {3}{1}"
+
+#: Actions.py:436
+#, python-format, python-brace-format
+msgid ""
+" {0}[ GETTING PRESSED KEYS ]{1}:\n"
+" {0}%s{1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 30]
```diff
@@ -0,0 +1,816 @@
...
+" {0}[ RECOLECTANDO TECLAS PRESIONADAS ]{1}:\n"
+" {0}%s{1}"
+
+#: Checks.py:21
+#, python-brace-format
+msgid "{0}Successful connection!{1}"
+msgstr "{0}Conexion Satisfactoria!{1}"
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 31]
```diff
@@ -0,0 +1,816 @@
...
+#, python-brace-format
+msgid ""
+"{1}\n"
+"        _  _ . ___  ___  ___ _  _  {0}___ _  _ ___{1}\n"
+"        |__| | ]  | ]  | |__ |\\ |  {0}|__ \\__/ |__{1}\n"
+"        |  | | ]__| ]__| |__ | \\|  {0}|__  ||  |__{1}\n"
+"\n"
+"                    {0}[{1}!{0}]{1} Network error. Verify your connection.\n"
+"\n"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 32]
```diff
@@ -0,0 +1,816 @@
...
+"        _  _ . ___  ___  ___ _  _  {0}___ _  _ ___{1}\n"
+"        |__| | ]  | ]  | |__ |\\ |  {0}|__ \\__/ |__{1}\n"
+"        |  | | ]__| ]__| |__ | \\|  {0}|__  ||  |__{1}\n"
+"\n"
+"                    {0}[{1}!{0}]{1} Error al conectar. Verifica tu conexion.\n"
+"\n"
+
+#: Checks.py:38
+msgid "[*] Ngrok Not Found !!"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Wrong

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 33]
```diff
@@ -0,0 +1,816 @@
...
+
+#: Checks.py:39
+msgid "[*] Downloading Ngrok..."
+msgstr "[*] Descargando Ngrok..."
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Okay

**Comment by sTiKyt** [comment]
@AgenorZ Also you have to fix this:
https://app.codacy.com/app/DarkSecDevelopers/HiddenEye/pullRequest?prid=3276710

**Comment by AgenorZ** [comment]
@sTiKyt Please check again

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 34]
```diff
@@ -0,0 +1,759 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: hiddeneye\n"
+"POT-Creation-Date: 2019-03-16 18:19-0400\n"
+"PO-Revision-Date: 2019-03-16 18:37-0400\n"
+"Last-Translator: AgenorZ <notemail@notemail.com>\n"
+"Language-Team: sTiKyt\n"
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Add yourself here.

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 35]
```diff
@@ -0,0 +1,759 @@
...
+"\n"
+" {0}[{1}2{0}]{1} Instagram Autoliker Phising (To Lure The Users)\n"
+"\n"
+" {0}[{1}3{0}]{1} Instagram Advanced Scenario (Appears as Instagram Profile)\n"
+"\n"
+" {0}[{1}4{0}]{1} Instagram Verified Badge Attack (Lure To Get Blue Badge){1} "
+"*[NEW]*\n"
+"\n"
+"{0}HiddenEye >>> {1}"
+msgstr ""
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Seems like being cut

File: locale/es/LC_MESSAGES/hiddeneye.po [hunk 36]
```diff
@@ -0,0 +1,759 @@
...
+"        {0}http://github.com/darksecdevelopers\n"
+"        {0}** CREADO POR:DARKSEC **\n"
+
+#: Actions.py:344
+#, python-brace-format
+msgid ""
+"-------------------------------\n"
+"{0}[ KEYLOGGER PROMPT ]{1}!! {0}\n"
+"-------------------------------"
+msgstr "{0}[ KEYLOGGER PROMPT ]{1}!! {0}"
```
Comment (sTiKyt) [inline locale/es/LC_MESSAGES/hiddeneye.po]: Not enough lines

**Comment by sTiKyt** [comment]
@wowficcion Congratulations xD

**Comment by wowficcion** [comment]
> @wowficcion Congratulations xD

Thanks hahaha, I'm newbie with Python

**Pull request merged**